vm/actor/src/builtin/miner/state.rs

```diff
@@ -1024,7 +1138,25 @@ mod tests {
             seal_proof_type: RegisteredSealProof::from(1),
             window_post_partition_sectors: 0,
         };
+        println!("miner info {:?}", &info);
         let bz = to_vec(&info).unwrap();
         assert_eq!(from_slice::<MinerInfo>(&bz).unwrap(), info);
     }
+
+    #[test]
+    fn miner_info_serde_json() {
+        let info = MinerInfo {
+            owner: Address::new_id(2),
+            worker: Address::new_id(3),
+            control_addresses: vec![Address::new_id(4), Address::new_id(5)],
+            pending_worker_key: None,
+            peer_id: PeerId::random().into_bytes(),
+            multi_address: vec![BytesDe(PeerId::random().into_bytes())],
+            sector_size: SectorSize::_2KiB,
+            seal_proof_type: RegisteredSealProof::from(1),
+            window_post_partition_sectors: 0,
+        };
+        let val = serde_json::to_string(&info);
+        println!("miner info {:?}", val);
+    }
```

Comment:
This test doesn't do anything, consider asserting the format? (You also shouldn't need the debug print for this)